unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -615,4 +653,39 @@ private void OnPlayerTeleportedToEditScene(Vector2Int coords)
     private void UpdateCatalogLoadingProgress(float catalogLoadingProgress) { initialLoadingController.SetPercentage(catalogLoadingProgress / 2); }
 
     private void UpdateSceneLoadingProgress(float sceneLoadingProgress) { initialLoadingController.SetPercentage(50f + (sceneLoadingProgress / 2)); }
+
+    private void OnUserProfileUpdate(UserProfile user)
+    {
+        userProfile.OnUpdate -= OnUserProfileUpdate;
+        updateLandsWithAcessCoroutine = CoroutineStarter.Start(CheckLandsAccess());
+    }
+
+    private IEnumerator CheckLandsAccess()
+    {
+        while (true)
+        {
+            UpdateLandsWithAccess();
+            yield return WaitForSecondsCache.Get(REFRESH_LANDS_WITH_ACCESS_INTERVAL);
+        }
```

Comment:
I am worried about to overload the servers with this, what do you think to deactivate this if you are already editing a scene/land?

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/563/commits/c600fe6b2326349349eb82200cc003b3aa31acf6. Thanks!